qa/L0_azure_storage/test.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
```

Comment:
Move this to L0_storage_azure to match naming convention of similar tests

Reply:
done.